common/common.go

```diff
@@ -45,8 +45,10 @@ func VersionCheck() (res *latest.CheckResponse, skip bool) {
 	return res, false
 }
 
-// IsGHError checks if the error is a github rate limit error
+// IsGHError checks  if any errror from github is returned
 func IsGHError(err error) bool {
-	_, ok := err.(*github.ErrorResponse)
-	return ok
+	_, rateLimit := err.(*github.RateLimitError)
```

Comment:
Does the test pass without the changes you made here?

Reply:
Yes the tests passed for these errors. But I have made few changes to generalise it to all Github API errors.